tests/queries/0_stateless/03254_timeseries_functions_various_arguments.reference

```diff
@@ -1,19 +1,22 @@
-[1,2,2,NULL,NULL,6,6,NULL,NULL,10,10,NULL]
-[1,2,2,NULL,NULL,6,6,NULL,NULL,10,10,NULL]
-[1,2,2,NULL,NULL,6,6,NULL,NULL,10,10,NULL]
-[1,2,2,NULL,NULL,6,6,NULL,NULL,10,10,NULL]
-[NULL,0.025,0.041666666666666664,0.03333333333333333,0.03333333333333333,0.1,0.1,0.1,NULL,0.1,0.1,0.1]
-[NULL,0.025,0.041666666666666664,0.03333333333333333,0.03333333333333333,0.1,0.1,0.1,NULL,0.1,0.1,0.1]
-[NULL,0.025,0.041666666666666664,0.03333333333333333,0.03333333333333333,0.1,0.1,0.1,NULL,0.1,0.1,0.1]
-[NULL,1.5,2.5,2,2,6,6,6,NULL,6,6,6]
-[NULL,1.5,2.5,2,2,6,6,6,NULL,6,6,6]
-[NULL,1.5,2.5,2,2,6,6,6,NULL,6,6,6]
-[NULL,0.1,0.1,0.1,0.1,0.1,0.1,0.1,NULL,0.1,0.1,0.1]
-[NULL,0.1,0.1,0.1,0.1,0.1,0.1,0.1,NULL,0.1,0.1,0.1]
-[NULL,0.1,0.1,0.1,0.1,0.1,0.1,0.1,NULL,0.1,0.1,0.1]
-[NULL,1,1,1,1,4,4,4,NULL,4,4,4]
-[NULL,1,1,1,1,4,4,4,NULL,4,4,4]
-[NULL,1,1,1,1,4,4,4,NULL,4,4,4]
+[1,2,NULL,NULL,NULL,6,NULL,NULL,NULL,10,NULL,NULL]
```

Comment:
For example, this line changed for the following reason. This line is output of the [command](https://github.com/ClickHouse/ClickHouse/blob/c6a14f1955bf5fe43bf2cb475d08aeb2b4070635/tests/queries/0_stateless/03254_timeseries_functions_various_arguments.sql#L17) which is an equivalent of the following command
```
SELECT timeSeriesResampleToGridWithStaleness(10, 120, 10, 10)([10, 20, 60,100]::Array(UInt32), [1, 2, 6, 10]::Array(Float32))
```

So before this PR the third value was calculated for window `[20, 30]` which contains one point `(20, 2)`, and now the third value is calculated for window `(20, 30]` which contains no points. That's how the third value changed from `2` to `NULL` here.